posthog-core/src/types.ts

```diff
@@ -128,13 +130,23 @@ export type PostHogAutocaptureElement = {
   [key: string]: any
 } // Any key prefixed with `attr__` can be added
 
+export enum Compression {
+  GZipJS = 'gzip-js',
+  Base64 = 'base64',
```

Comment:
we should add `gzip` as well since might be needed for Node and React Native

Reply:
GZipJS and GZIP are aliases to each other

Reply:
true that, just found this here https://github.com/PostHog/posthog/blob/bcc624b7150124354edb4cd1f603cf73ce1dbfc9/posthog/api/decide.py#L118-L119